tuning/trackers/filelogging_tracker.py

```diff
@@ -80,7 +80,10 @@ def __init__(self, tracker_config: FileLoggingTrackerConfig):
                 which contains the location of file where logs are recorded.
         """
         super().__init__(name="file_logger", tracker_config=tracker_config)
-        self.logger = logging.get_logger("file_logging_tracker")
+        # Configure log level
+        LOGLEVEL = os.environ.get("LOG_LEVEL", "WARNING").upper()
+        logging.basicConfig(level=LOGLEVEL)
```

Comment:
`logging.basicConfig()` sets the log level for the root logger. But you can set individual loggers level by running `self.logger.setLevel(level)`

Reply:
There are a few of these to fix, not going to comment in each file

Reply:
I don't think you even have to set log level for every file, you only have to set it once in the sft_trainer.py after parsing user's input right?

In each file, you just need to get a logger by doing:
```
import logging
logger = logging.getLogger(__name__)
```
and every time we log something, we just use that `logger`:

```
logger.info("Log this info")
```

Reply:
@aluu317 Yes I think it makes sense.
I made changes and used `logger = logging.getLogger("sft_trainer_main")` everywhere.

Reply:
I agree that having each file set their own logging level helps to ensure that the logging level is set if a file is called outside of sft_trainer.py. I do agree that using `logging.getLogger(__name__)` is best practice and it is unlikely these files will be called outside of sft_trainer.py so I think going with Angel's suggestion works